CHANGELOG.md

```diff
@@ -1,406 +1,34 @@
-  - added DEVELOPER.md for SohoXi Angular Component developers
-  - copy SohoXi CSS files to `src/assets` on build
-  - added _lint_ and _code coverage_ to _unit-test_ npm script
-  - automatically include SohoXi typings when consuming NPM package.
-* 2016-11-04 - BTH - PR-XXX
-  * Added module support for _@infor/sohoxi-angular_.
-* 2016-10-13 - BTH - PR-XXX
- * added typings for listview and splitter.
- * SohoListViewComponent now uses Soho Control defaults for 'options'
+## <a name="version-master">master</a>
 
-### Breaking Changes
+### Fixes
 
-* 2018-05-08 - CRL - NPM packages changes for Enterprise Components for Angular
-The `angular.json` file requires changes to reference the new packages, spefifically:
-```json
-"assets": [
-  {
-    "glob": "**/*",
-    "input": "./node_modules/ids-enterprise/dist/css",
-    "output": "./assets/ids-enterprise/css"
-  }
-]
-```
-```json
-"scripts": [
-    "./node_modules/jquery/dist/jquery.js",
-    "./node_modules/ids-enterprise/dist/js/sohoxi.js",
-    "./node_modules/ids-enterprise/dist/js/cultures/en-US.js",
-    "./node_modules/ids-enterprise/dist/js/d3.v4.js"
-  ]
-```
-Applications will be need to change all import lines that specify **@infor/sohoxi-angular** to **ids-enterprise-ng**. For example:
-```typescript
-import { SohoButton } from 'ids-enterprise-ng';
-```
-**package.json** must be updated to import **ids-enterprise** and **ids-enterprise-ng**.
-```json
- "ids-enterprise": "^4.7.0-dev.20180511",
- "ids-enterprise-ng": "^4.7.0-dev.20180511",
-```
-**index.html** may required updates if assets are being included, for example:
-```html
-<link rel="stylesheet" id="stylesheet" href="/assets/ids-enterprise/css/light-theme.css" type="text/css">
-```
-**tsconfig.json** must be updated to point to the new `ids-enterprise-ng` package.  Add `node_modules/ids-enterprise-ng/**/*` to the `include` property, as follows:
+- `[Component]` Example note item for future release. `INI` ([#88](https://github.com/infor-design/enterprise-ng/issues/88))
 
-```json
-"include": [
-  "src/**/*",
-  "node_modules/ids-enterprise-ng/**/*"
-]
-```
-* 2018-05-08 - BTH - Upgraded to @angular 6.0.0 and @angular/cli 6.0.0  
-It is recommended that consumers of the the **ids-enterprise-ng** components also upgrade their applications to version 6 of @angular and @angular/cli.  The easiest way to do this is to follow the instructions at https://update.angular.io/.
+## <a name="version-4.7.0">4.7.0</a>
 
-* 2018-05-07 - PWP - SOHO-7923: Refactored several components and directives to run outside angular
-changed soho-busyindicator.directive.ts' 'close' event to 'complete' to match the jquery component - busyindecator.js  
+### <a name="version-4.7.0-features">Features</a>
 
-* 2018-04-12 - MAF - SOHO-7711: 4.0 File upload component refactor to support directive usage - changed to <input soho-fileupload /> from <soho-fileupload>
+- `[General]` Upgraded to latest angular/cli release (6 `BTH`
+- `[Tag]` Added Tag support `BTH`
```

Comment:
is there supposed to be a closing parentheses after (6 ?

wasn't part of the original comment, so not sure